packages/core/src/BackgroundImage/BackgroundImage.tsx

```diff
@@ -50,10 +51,11 @@ export interface IBackgroundImageProps
   image?: string
   borderRadius?: string
   rounded?: string
+  backgroundPosition?: string
```

Comment:
If you follow this example, you can define an array value for use in the propTypes at line 42, then use that array to create a type for use in the props interface at 54. The type for `backgroundPosition` must reflect the allowed values, not any string.

https://www.typescriptlang.org/play?#code/MYewdgzgLgBANgS2jAvDA2gcgIaYDQyYBG+hwmAujNhDKJFANwwD0LMAKgMoDMAdABYYEAJ5go2AB4AoKCIAOAUxgA5RYoAmmgKpgE4DguVo5SkADN4SKOjABXALZFFAJwrM2hXAB9imX+SM0kA